dataset.py

```diff
@@ -137,3 +142,91 @@ def yearly_255_trees_partitioned(reference_csv_filename: str) -> PartitionedData
   print_split(partition_data)
   return partition_data
 
+#Utility function for randomly sampling a point around a sample site
+def is_valid_point(lat: float, lon: float, reference_isocape: raster.AmazonGeoTiff):
+  return True if raster.get_data_at_coords(reference_isocape, lon, lat, 0) else False
+
+# Pick a random point around (lat, lon) within max_distance_km. If edge_only is
+# true, only pick points exactly max_distance_km away from (lat, lon).
+def random_nearby_point(lat, lon, max_distance_km, edge_only=False):
```

Comment:
Could we specify the parameter data types of the function like you do for is_valid_point?